packages/client/src/components/set/index.css

```diff
@@ -81,4 +81,8 @@
 
 .viewport {
     min-height: 80vh;
+}
+
+.set-icon {
+    filter: invert(0.5) sepia(1) saturate(3) hue-rotate(0deg);
```

Comment:
https://stackoverflow.com/questions/24933430/img-src-svg-changing-the-styles-with-css

^ never seen this before but it does the job 🎉